### DISCUSSION TITLE
[SOLVED] Loading image for OpenGl from uns. char.

### DISCUSSION THREAD

**Discussion by Stormfinger** [ImageMagick/ImageMagick#5234] [Help]
i dont know if its proper place to ask but i have problems with image transparency once i convert png to h. currently i use SOIL2 library for loading image from unsigned char. do you know of any better lib to load h file into OpenGL texture. Or is there a problem with transparency in h file?

**Comment by snibgo** [comment]
See https://imagemagick.org/script/defines.php . The PNM format doesn't record opacity. PNG and GIF can. For example:
```
magick in.png -define h:format=GIF out.h
```

**Comment by Stormfinger** [comment]
thank you.. i just couldnt find this information on the web.
